examples/helper.py

```diff
@@ -76,7 +76,13 @@ def get_commandline(server=False, description=None, extras=None, cmdline=None):
             help="set number of slaves, default is 0 (any)",
             default=0,
             type=int,
-            nargs="+",
+        )
+        parser.add_argument(
+            "--zero-mode",
```

Comment:
Zero mode is no longer a supported parameter.

Reply:
That is to say it is in the process of being removed.

Reply:
Removed